db/migrate/20161206131603_create_drivers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDrivers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :drivers do |t|
+      t.string :name
+      t.string :phone
```

Comment:
@domenic99 are you sure that we could have any drivers without a phone in our database?